nodeJS/models/news-model.js

```diff
@@ -0,0 +1,23 @@
+const logger = require("../logger");
+const mongoose = require("../database");
+
+const newsJSON = require("../consts/news.json");
+
+const schema = {
+  ID: { type: mongoose.SchemaTypes.String, required: true },
+  name: { type: mongoose.SchemaTypes.String, required: true }
+};
+
+const collectionName = "news";
+const newsSchema = mongoose.Schema(schema);
+const News = mongoose.model(collectionName, newsSchema);
+
+News.deleteMany({}, () => console.log("collection cleared"));
+
+News.create(newsJSON, err => {
```

Comment:
It is a good practice to notify that some data were created successfully.

Reply:
logging added